### PR TITLE
doc: update release dates

### PR DESCRIPTION
**Pull request by BethGriggs**

### CODE REVIEW

File: schedule.json [hunk 1]
```diff
@@ -52,14 +52,24 @@
   },
   "v11": {
     "start": "2018-10-23",
-    "maintenance": "2019-04-01",
-    "end": "2019-06-30"
+    "maintenance": "2019-04-16",
```
Comment (BethGriggs) [inline schedule.json]: I set 2019-04-16 as the date for maintenance as that is the date of the last planned v11.x release

**Comment by BethGriggs** [comment]
@nodejs/release PTAL at the estimated dates for the next few releases. Also, could someone please link me to the tool that generates the timeline image?

Refs: https://github.com/nodejs/Release/issues/431? /cc @sam-github 

@jasonkarns, I've set the schedule.json to put v11.x into "_maintenance_" on 2019-04-22, and v12.x to start on 2019-04-23. Does this work?

**Comment by richardlau** [comment]
> 
> 
> Also, could someone please link me to the tool that generates the timeline image?
> 

@BethGriggs https://github.com/nodejs/lts-schedule, See also https://github.com/nodejs/Release/pull/380#issue-226387243.

File: schedule.json [hunk 2]
```diff
@@ -52,14 +52,24 @@
...
     "start": "2019-04-23",
     "lts": "2019-10-22",
     "maintenance": "2021-04-01",
-    "end": "2022-04-01",
-    "codename": ""
+    "end": "2022-04-01"
+  },
+  "v13": {
+    "start": "2019-10-22",
+    "maintenance": "2020-04-01",
```
Comment (targos) [inline schedule.json]:
```suggestion
    "maintenance": "2020-04-20",
```

File: schedule.json [hunk 3]
```diff
@@ -52,14 +52,24 @@
...
-    "end": "2019-06-30"
+    "maintenance": "2019-04-22",
+    "end": "2019-06-01"
   },
   "v12": {
     "start": "2019-04-23",
     "lts": "2019-10-22",
     "maintenance": "2021-04-01",
-    "end": "2022-04-01",
-    "codename": ""
```
Comment (MylesBorins) [inline schedule.json]: do we need to keep the codename here?

```suggestion
 "codename": "TBD"
```

**Comment by jasonkarns** [comment]
> I've set the schedule.json to put v11.x into "_maintenance_" on 2019-04-22, and v12.x to start on 2019-04-23. Does this work

@BethGriggs that's perfect, thanks!

**Comment by BethGriggs** [comment]
Screenshot 2019-04-15 at 16 42 51

@nodejs/release, I have updated the timeline SVG - does this look ok?

**Comment by MylesBorins** [comment]
LGTM

**Pull request merged**